### PR TITLE
CORE-18507: Add UnamangedKeyStatus that will serve both key status and key rotation

### PR DESCRIPTION
**Pull request by anien**
* Adds avro schema that serves both key status and key rotation status requests. 
* This schema is stored in the db.

### CODE REVIEW

File: data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key.status/UnmanagedKeyStatus.avsc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "fields": [
+    {
+      "name": "rootKeyAlias",
+      "type": "string",
+      "doc": "Alias of an unmanaged key that we are rotating away from."
+    },
+    {
+      "name": "tenantId",
+      "type": "string",
+      "doc": "GUID of the tenantId or clusterDB."
```
Comment (simon-johnson-r3) [inline data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key.status/UnmanagedKeyStatus.avsc]: Is this really a GUID? I thought it was either the short hash of the vnode or a string from a list defined in Corda? Also do we need this if it's the key?

File: data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key.status/UnmanagedKeyStatus.avsc [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "type": "record",
+  "name": "UnmanagedKeyStatus",
+  "namespace": "net.corda.data.crypto.wire.ops.key.status",
+  "doc": "Defines the key status and key rotation status data stored in the database.",
```
Comment (simon-johnson-r3) [inline data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key.status/UnmanagedKeyStatus.avsc]: I wouldn't use the word database here as there are future plans to sit the statemanager on other things.

File: data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key.status/UnmanagedKeyStatus.avsc (hunk 1 above)
Comment (anien) [reply data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key.status/UnmanagedKeyStatus.avsc]: You are right. It's not a GUID. 
I put it here to have the relevant data in one place, but you are right, `tenantId` will be a key, so we don't necessarily need it here. I will remove it and we can always put it back if we find the need for it.

File: data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key.status/UnmanagedKeyStatus.avsc (hunk 2 above)
Comment (anien) [reply data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key.status/UnmanagedKeyStatus.avsc]: Good point. Removed.

**Review by simon-johnson-r3 — APPROVED** [review]

**Pull request merged**